src/ExpressionEvaluator/Package/ExpressionEvaluatorPackage.csproj

```diff
@@ -41,48 +48,76 @@
       <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup</IncludeOutputGroupsInVSIX>
       <IncludeOutputGroupsInVSIXLocalOnly>DebugSymbolsProjectOutputGroup</IncludeOutputGroupsInVSIXLocalOnly>
       <ForceIncludeInVSIX>true</ForceIncludeInVSIX>
+      <Ngen>true</Ngen>
```

Comment:
Ah, I had missed a compressed line and it looked like this was `ServicesVisualStudioImpl`